libfdt/fdt_overlay.c

```diff
@@ -444,8 +444,8 @@ static int overlay_fixup_phandle(void *fdt, void *fdto, int symbols_off,
 			return -FDT_ERR_BADOVERLAY;
 
 		ret = overlay_fixup_one_phandle(fdt, fdto, symbols_off,
-						path, path_len, name, name_len,
-						poffset, label);
+										path, path_len, name, name_len,
+										poffset, phandle);
```

Comment:
These lines are oddly over-indented in the revised version, can you fix this up please.